src/realm/object-store/object_schema.hpp

```diff
@@ -96,6 +90,25 @@ class ObjectSchema {
 private:
     void set_primary_key_property() noexcept;
 };
+
+inline std::ostream& operator<<(std::ostream& o, ObjectSchema::TableType table_type)
+{
+    switch (table_type) {
+        case ObjectSchema::TableType::TopLevel:
+            o << "TopLevel";
+            break;
+        case ObjectSchema::TableType::Embedded:
+            o << "Embedded";
+            break;
+        case ObjectSchema::TableType::TopLevelAsymmetric:
+            o << "TopLevelAsymmetric";
+            break;
+        default:
+            o << "Unknown table type";
```

Comment:
If this isn't just going to be REALM_UNREACHABLE, do we want to output the value as an integer here for debugging purposes?

Reply:
good point